blog-posts/thoughs-on-collaborative-work.md

```diff
@@ -0,0 +1,79 @@
+
+
+---
+
+Most of us developers, despite being more or less aware of the benefits of doing more work together, work on our own most of the time. Why? What can we do about it? Let's dive into it!
+
+## Is it just about programming?
+
+Short answer, no, it’s not. As software developers we don’t just write code, we also play a crucial role in actively shaping, challenging and clarifying requirements. And this happens in many different flavours: emails, issue tracking systems, chat tools, team meetings, inter-team meetings... Every area of our work can benefit if we approach it in a more collaborative way.
+
+## The good, the bad and the ugly. Or is it just good?
+
+What is the greatest hesitance when thinking about working more with other colleagues? Well, for most of us it may feel cumbersome in terms of time consumption and efficiency. But if we dig deeper, we will probably find out that this hesitance has a lot to do with the well-known [impostor syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome). Often times we might feel like we are not good enough. The need to ask for help, the idea of someone watching you work or seeing the uncertainty with which you approach problems or your lack of knowledge in some areas, etc. is frightening for most of us. Literally “stage fright”.
+
+These fears are common in developers regardless of their degree of expertise, but the truth is that we are all learning all the time, and the more we interact with others, the more we learn. I am not an exception, but I decided to open myself more to collaborative work for three main reasons:
+
+- I was not happy with the amount and quality of collaborative work in my teams
+- I consider these beliefs mostly as prejudices
+- Every time I was approaching work in a more collaborative way, even though feeling unnatural, it showed to be quite powerful
+
+Don't get me wrong, I am not talking about overdoing it, but about finding the balance, the sweet spot where collaborative work produces a positive impact in our working life. In order for that to happen, it is crucial to accumulate experiences and get a feeling of which collaboration paradigm is better to use for each use case. Daily meeting? Knowledge sharing session? Second opinion? Think of a toolbox for team collaboration, this is exactly what is offered to you. Every person, every situation, every topic is different, that's what makes this journey exciting and why you want to have a variety of options and adapt each time.
+
+## In practice 
+
+In this section I will present you four fantastic, experience-proved ways to foster collaboration in your team. As you will realise, the common factors of any kind of collaborative work are: clear goals, preparation, focus, constantly challenging yourself and each other’s thoughts and a constructive attitude. 
+
+Don't limit yourself to this list, but use it also as inspiration to find your own ways!
+
+#### The powerful start 🚀
```

Comment:
h2 changes to h4. I can't find my previous comment right now but I'm sure I commented on that. That's a wrong outline and this must be h3

Reply:
I also didn't see that comment. I updated the headlines, thanks for noticing.